src/playground/sample_data_ingestion/jobs/refresh_job.py

```diff
@@ -0,0 +1,145 @@
+# opensearch-py==2.0.0
+from opensearchpy import OpenSearch
+
+# Standard libraries
+import sys
+from os import environ, listdir, path
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from sample_data_indices.sample_data_indices import SampleDataIndex
+from datetime import date, timedelta, datetime
+from shutil import copyfileobj
+from os import remove
+from math import ceil
+from json import load
+import gzip
+
+
+# Array in which unzipped files will be removed
+file_removal_array = []
+
+
+# Utility function for unzipping files
+def unzip(filename):
+  if type(filename) is not str or (type(filename) is str and ".json" not in filename):
```

Comment:
I saw similar check cross all your tools, could you please create a small function and add it to common library?

Reply:
Done, all pertinent logic has been moved to `sample_data_commons/utils.py`.